src/screens/StudentGroup.tsx

```diff
@@ -118,21 +167,49 @@ const StudentGroup: React.FC = () => {
           />
         </Box>
 
-        {/* <Button alignSelf="flex-end" marginBottom=".3rem" onClick={onOpen}>
-          <Icon as={BiUserPlus} marginRight=".5rem" fontSize="2rem" />
-          Add Students
-        </Button> */}
+        {/* <Box width="85vw" maxWidth="40rem" maxHeight="40rem" gridArea="pic"></Box> */}
+        <Flex
+          direction="column"
+          justify="flex-start"
+          padding={{ base: 'none', md: '0 2rem' }}
+          justifySelf="start"
+          alignItems="flex-start"
+          minHeight="5.5rem"
+          gridArea="fact"
+        >
+          {studentGroupDocument?.selectedStudent?.studentInfo?.selectedFact && (
+            <Flex direction="column" justify="flex-start" alignItems="flex-start" lineHeight="28px">
+              <Heading
+                as="h2"
+                fontSize={{ base: '1.1rem', md: '1.6rem', lg: '2rem' }}
+                color="var(--main-color-medium)"
+                width="100%"
+              >
+                {studentGroupDocument?.selectedStudent.studentInfo.selectedFact.title}
+              </Heading>
+
+              <Heading as="h2" fontSize={{ base: '1.5rem', md: '2.3rem', lg: '3rem' }} letterSpacing=".05em">
+                {studentGroupDocument?.selectedStudent.studentInfo.selectedFact.value}
```

Comment:
`?` shouldn't be needed right? we can't get here if line 180 is falsey.

Reply:
Good point, I'll get rid of that.